src/Sentry.Unity/Integrations/IAppDomain.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Sentry.Unity.Integrations
+{
+    internal interface IAppDomain

```

Comment:
Should this be `IApplication` instead?

The .NET SDK is creating an abstraction over a class called `AppDomain` that's why `IAppDomain` there.